.github/workflows/wiki_notification.yml

```diff
@@ -0,0 +1,45 @@
+name: Wiki Notification
+
+on: [gollum,workflow_dispatch]
+  
+
+jobs:
+  run_script:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: get informations from api and prepare email messages
+      run: |
+        response=$(curl -s https://api.github.com/repos/cgal/cgal/events)
+        eventcount=$(echo "$response" | jq '. | length')
+        for ((i=eventcount; i>=0; i--))
+        do
+            if [ "$(echo "$response" | jq -r '.['$i'].type')" == "GollumEvent" ];
+            then
+                event=$(echo "$response" | jq '.['$i']')
+            fi
+        done
+        actor=$(echo "$event" | jq '.actor.login' | tr -d '"')
+        count=$(echo "$event" | jq  '.payload.pages | length')
+        date=$(echo "$event" | jq '.created_at' | tr -d '"')
+        echo -e "Subject:Cgal wiki has been updated \n" >> messages.txt
+        for ((i=0; i<count; i++)); do
+          action=$(echo "$event" | jq '.payload.pages['$i'].action' | tr -d '"')
+          wiki_page=$(echo "$event" | jq '.payload.pages['$i'].title' | tr -d '"')
+          page_url=$(echo "$event" | jq '.payload.pages['$i'].html_url' | tr -d '"')
+          echo "Page $i" >> messages.txt
+          echo "Actor : $actor" >> messages.txt
+          echo "Action : $action" >> messages.txt
+          echo "Wiki Page : $wiki_page" >> messages.txt
+          echo "Page URL : $page_url" >> messages.txt
+          echo "History : $page_url/_history" >> messages.txt
+          echo "Date : $date" >> messages.txt
+        done
+    - name: install ssh keys
+      run: |
+        install -m 600 -D /dev/null ~/.ssh/id_rsa
+        echo "${{ secrets.SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
+        ssh-keyscan -H ${{ vars.SSH_HOST }} > ~/.ssh/known_hosts
+    - name: send email via ssh
+      run: |
+        cat messages.txt | ssh ${{ secrets.SSH_USER }}@${{ vars.SSH_HOST }} "/sbin/sendmail -t ${{ vars.SEND_EMAIL_TO }}"
```

Comment:
The objective of these two steps is to send an email. I would like you to create a [reusable workflow](https://docs.github.com/en/actions/using-workflows/reusing-workflows), dedicated to the sending of email.

What is more, the names of the variables and secrets should be more specific, so that they can be distinguished from the other variables and secrets:
```suggestion
    - name: install ssh keys
      run: |
        install -m 600 -D /dev/null ~/.ssh/id_rsa
        echo "${{ secrets.CGAL_SEND_EMAIL_SSH_PRIVATE_KEY }}" > ~/.ssh/id_rsa
        ssh-keyscan -H ${{ vars.CGAL_SEND_EMAIL_SSH_HOST }} > ~/.ssh/known_hosts
    - name: send email via ssh
      run: |
        cat messages.txt | ssh ${{ secrets.CGAL_SEND_EMAIL_SSH_USER }}@${{ vars.CGAL_SEND_EMAIL_SSH_HOST }} "/sbin/sendmail -t ${{ vars.SEND_EMAIL_TO }}"
```

The following will be transformed into "inputs" of the reusable workflow:
 - `messages.txt`, and
 - `vars.SEND_EMAIL_TO`.


The goal of that creating of a reusable workflow is... to be able to reuse it! :smile: 

There are other events for which we would like to be able to receive email notifications. For exemple, we would like to get Git push events, using [`git-multimail`](https://github.com/git-multimail/git-multimail). That will be a later task.